text/0000-test-framework-binnacle.md

```diff
@@ -0,0 +1,373 @@
+- Start Date: 2020-01-07
+- Tracking: (leave this empty)
+- RFC PR: [kadalu/rfcs#6](https://github.com/kadalu/rfcs/pull/6)
```

Comment:
Please rename this, and file name to appropriate name and feel free to push after another 24hrs.

/approved